src/Common/src/Internal/NativeFormat/NativeFormatWriter.cs

```diff
@@ -538,6 +600,478 @@ public override int GetHashCode()
 #else
     internal
 #endif
+    class MethodNameAndSigSignature : Vertex
+    {
+        Vertex _methodName;
```

Comment:
All fields in the concrete Vertex types should be `private readonly`; or at least `private`.